lib/queue.js

```diff
@@ -725,6 +734,7 @@ class Queue extends Emitter {
       .then((results) => {
         const numRaised = results[0], nextOpportunity = results[1];
         if (numRaised) {
+          // not documented
```

Comment:
heh yeah

This seems unrelated to the overarching PR, and maybe we could document it instead of leaving it more explicitly undocumented.